active-directory-java-native-headless-v2/CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to Azure samples
+
+Thank you for your interest in contributing to Azure samples!
+
+## Ways to contribute
+
+You can contribute to [Azure samples](https://azure.microsoft.com/documentation/samples/) in a few different ways:
+
+- Submit feedback on [this sample page](https://azure.microsoft.com/documentation/samples/active-directory-java-native-headless/) whether it was helpful or not.  
+- Submit issues through [issue tracker](https://github.com/Azure-Samples/active-directory-java-native-headless/issues) on GitHub. We are actively monitoring the issues and improving our samples.
```

Comment:
These links would need to be updated to this repo

Reply:
updated